### PR TITLE
Release related changes for 0.6.0

### PR DESCRIPTION
**Pull request by ankitjavalkar**

### CODE REVIEW

**Comment by maheshgudi** [comment]
LGTM

File: CHANGELOG.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+=== 0.6.0 (unreleased) ===
+
+* Added a course code field that can be used to search for a code.
```
Comment (prathamesh920) [inline CHANGELOG.txt]: I think it should be "to search a course"

File: CHANGELOG.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* Update the documentation to describe the course code feature
```
Comment (prathamesh920) [inline CHANGELOG.txt]: Updated

File: CHANGELOG.txt [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+* Added the facility to allow moderators to see and download uploaded assignments.
+* Fixed a bug that prevented teachers of a course from editing Question papers.
+* Added a start time and end time for enrollment of a course.
+* Added the facility to allow moderators to edit the details of a course.
+* Added a test_settings.py file to avoid using the committed migrations and rebuild database from scratch while running tests.
+
+=== 0.5.0 (22-03-2017) ===
+
+* Fixed a bug that prevented question tags from being saved when saving questions.
+* Add new question types: 'String Type', 'Integer Type' and 'Float Type'.
```
Comment (prathamesh920) [inline CHANGELOG.txt]: Added

**Comment by maheshgudi** [comment]
Closing this PR due to Ankit's absence. Will reopen with necessary changes made.

**Pull request closed without merging**